### DISCUSSION TITLE
Power management

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#221] [Bug Reports]
Hello
Testing lcd display feature : ok perfect job
Problem on soft shutdown, which is only a standby mode and not a full power off., thus the display remain lighting even we shutdown the device.

**Comment by rern** [comment]
The LCD display gets 5V supply from GPIO which is still active when "power off".

**Reply by Leobo71** [reply]
Will net a shutdown with switching off the backlight LED
